cg/constants/scout_upload.py

```diff
@@ -14,7 +14,7 @@
 
 BALSAMIC_CASE_TAGS = dict(
     snv_vcf={"vcf-snv-clinical"},
-    sv_vcf={"vcf-sv-clinical"},
```

Comment:
do we have anything that is under this tag that wont be delievered now?

Reply:
Nope, there never was anything under the flag.

Reply:
It's back in the new version of balsamic, it's the filtered vcf.